examples/gnrc_border_router/Makefile.esp.conf

```diff
@@ -0,0 +1,3 @@
+# Set the SSID and password of your WiFi network here
+CFLAGS += -DESP_WIFI_SSID=\"Your WiFi name\"
+CFLAGS += -DESP_WIFI_PASS=\"Your secure password\"
```

Comment:
We should _really_ unify this for more than ESP ;-) (but maybe in a separate PR).